### ISSUE TITLE
pmempool check CLI: consistency check fails on pool set with remote replica

### ISSUE THREAD

**Issue by kkajrewicz** [pmem/issues#290]
Checking consistency of the pool based on pool set file with remote replica using pmempool check CLI command fails:
```
#pmempool check pool.set
error: Invalid argument
error: repairing failed
```
 Dbg. logs:
```
<libpmempool>: <1> [set.c:182 util_remote_load] remote replication is not available
<libpmempool>: <1> [set.c:2289 util_pool_open_nocheck] the pool set requires a remote replica, but the 'librpmem.so.1' library cannot be loaded
<libpmempool>: <3> [pool.c:501 pool_data_free]
<libpmempool>: <3> [check_util.c:125 check_data_free]
error: Invalid argument
error: repairing failed
<libpmempool>: <3> [libpmempool.c:72 libpmempool_fini]
```

Consistency check works when it is done by pmemobj_check() function.

Found in 1.1-714-g8b0b468

**Comment by kkajrewicz** [comment]
How was it fixed? Still able to reproduce issue on 1.1-751-g222fe0c

**Comment by janekmi** [comment]
It is not the same behavior. Log shows reason why provided arguments are considered invalid:
(newlines added for emphasis)
```
<libpmempool>: <10> [set.c:880 util_poolset_parse] REMOTE REPLICA node address 'localhost' pool set descriptor 'testset_remote'
<libpmempool>: <3> [set.c:731 util_parse_add_remote_replica] setp 0x7ffd37ab5160 node_addr localhost pool_desc testset_remote
<libpmempool>: <3> [set.c:668 util_parse_add_replica] setp 0x7ffd37ab5160
<libpmempool>: <3> [set.c:634 util_parse_add_part] set 0xa89c90 path (null) filesize 8192
<libpmempool>: <4> [set.c:916 util_poolset_parse] set file format correct (/tmp/node1/test_obj_rpmem_basic_integration/testset_local)
<libpmempool>: <3> [set.c:720 util_poolset_set_size] pool size set to 8388608

<libpmempool>: <1> [pool.c:311 pool_params_parse] poolsets with remote replicas are not supported

<libpmempool>: <3> [pool.c:516 pool_data_free] 
<libpmempool>: <3> [check_util.c:125 check_data_free] 
<libpmempool>: <3> [libpmempool.c:81 libpmempool_fini] 
<libpmempool>: <3> [set.c:104 util_remote_fini] 
```

So libpmempool validates arguments, logs what is the issue, sets errno and closes gracefully.
I would prepare pull request with update to the documentation because I think it is the only thing that is missing.
